hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/ClientJobProxy.java

```diff
@@ -202,7 +218,7 @@ public Void get() throws InterruptedException, ExecutionException {
 
         @Override
         public Void get(long timeout, @Nonnull TimeUnit unit)
-                throws InterruptedException, ExecutionException, TimeoutException {
+            throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
this looks weird

Reply:
Wrong setting for "continous indent" in IntelliJ, fixed it.